### PR TITLE
[ci skip] update 4.x release doc

### PR DESCRIPTION
**Pull request by tbugfinder**
please edit as needed.

### CODE REVIEW

File: docs/modules/getting-started/pages/index.adoc [hunk 1]
```diff
@@ -67,17 +67,18 @@ A minor update (consuming changes/bugfixs on a release branch such as 3.x) is, f
...
-=== https://github.com/orgs/opendevstack/projects/10[4] (Target date: Early 2021)
+=== https://github.com/orgs/opendevstack/projects/14[4.2] (Target Date: End-of-Feb 2022)
+- See project page for details.
 
-Planned work (subject to change):
+=== https://github.com/orgs/opendevstack/projects/10[4] (November 2021)
 
 - OpenShift 4 support (keeping 3.11 compatibility)
 - Support for Kubernetes-native Deployment resources
 - Support for Helm 3
```
Comment (michaelsauter) [inline docs/modules/getting-started/pages/index.adoc:79]: Both `Support for Kubernetes-native Deployment resources` and `Support for Helm 3` work ONLY for the component pipeline.

File: docs/modules/getting-started/pages/index.adoc [hunk 2]
```diff
@@ -67,17 +67,18 @@ A minor update (consuming changes/bugfixs on a release branch such as 3.x) is, f
...
 - Deployment to multiple D/Q/P namespaces/clusters
```
Comment (michaelsauter) [inline docs/modules/getting-started/pages/index.adoc:80]: This hasn't been implemented.

File: docs/modules/getting-started/pages/index.adoc [hunk 3]
```diff
@@ -67,17 +67,18 @@ A minor update (consuming changes/bugfixs on a release branch such as 3.x) is, f
...
-- New/Reworked machine learning quickstarter
+- dropped machine learning quickstarter
 - Integration with Aqua Security
 - Implement health checks for quickstarters
 - Provisioning App: Support config of multiple identity providers and enable new UI by default
```
Comment (michaelsauter) [inline docs/modules/getting-started/pages/index.adoc:84]: This hasn't been implemented as far as I know.

**Review by michaelsauter — COMMENTED** [review]
Sorry I think I can't push into your repo so I just added some comments.

**Comment by tbugfinder** [comment]
> Sorry I think I can't push into your repo so I just added some comments.

Isn't this the purpose of `Allow edits and access to secrets by maintainers`?

The file was updated - please check.

**Review by michaelsauter — APPROVED** [review]

**Pull request merged**